### ISSUE TITLE
the initial lr value is higher than target lr value

### ISSUE THREAD

**Issue by developer0hye** [ildoonet/pytorch-gradual-warmup-lr#7]
Thank you for sharing great works!

The initial LR value seems to be larger than I expected.

# code

```python
v = torch.zeros(10)
optim = torch.optim.SGD([v], lr=1e-2)
scheduler = GradualWarmupScheduler(optim, multiplier=8, total_epoch=10)

 for epoch in range(1, 20):
    scheduler.step(epoch)
    print(epoch, optim.param_groups[0]['lr'])
```

# printed results

1 0.017
2 0.024
3 0.031000000000000003
4 0.038
5 0.045
6 0.052000000000000005
7 0.059000000000000004
8 0.066
9 0.073
10 0.08
11 0.08
12 0.08
13 0.08
14 0.08
15 0.08
16 0.08
17 0.08
18 0.08
19 0.08

As you can see, the initial lr value (0.017) is higher than target lr value (0.01).

This result is right ??

**Comment by ildoonet** [comment]
@developer0hye Yes, it is expected behavior. 

But I suggest you to use zero-based epoch number.

```
scheduler.step(epoch - 1)
```

**Comment by developer0hye** [comment]
@ildoonet 
Thanks!
